lib/typescript/class-names-to-type-definition.ts

```diff
@@ -34,15 +42,20 @@ const isValidName = (className: ClassName) => {
 
 export const classNamesToTypeDefinitions = (
   classNames: ClassNames,
-  exportType: ExportType
+  exportType: ExportType,
+  quoteType: QuoteType = "single"
 ): string | null => {
   if (classNames.length) {
     let typeDefinitions;
 
     switch (exportType) {
       case "default":
         typeDefinitions = "export interface Styles {\n";
-        typeDefinitions += classNames.map(classNameToInterfaceKey).join("\n");
+        typeDefinitions += classNames
+          .map((className: ClassName) => {
```

Comment:
nits: 

- we could probably remove `ClassName` here since `classNames` _should_ already be properly typed _(if not, that should be fixed)_. And if it's `any` somehow, we should then get a type error about no implicitly anys
- can drop the `{ return ... }` for just `...`, `map(className => classNameToInterfaceKey(...))`